_pyinstaller_hooks_contrib/stdhooks/hook-saml2.py

```diff
@@ -0,0 +1,27 @@
+# ------------------------------------------------------------------
+# Copyright (c) 2024 PyInstaller Development Team.
+#
+# This file is distributed under the terms of the GNU General Public
+# License (version 2.0 or later).
+#
+# The full license is available in LICENSE, distributed with
+# this software.
+#
+# SPDX-License-Identifier: GPL-2.0-or-later
+# ------------------------------------------------------------------
+
+# Hook for https://github.com/IdentityPython/pysaml2
+from PyInstaller.utils.hooks import collect_data_files, copy_metadata, collect_submodules
+
+datas = copy_metadata("pysaml2")
+
+# The library contains a bunch of XSD schemas that are loaded by the code:
+# https://github.com/IdentityPython/pysaml2/blob/7cb4f09dce87a7e8098b9c7552ebab8bc77bc896
+# /src/saml2/xml/schema/__init__.py#L23
```

Comment:
Can we keep these URLs in single line? Even if it exceeds the nominal line length limit of 120 that we use, I think linter has enough decency to not make a fuss about it when it is a single "word".

Reply:
Of course. I added a commit with a fix. I was not sure if you prefer new commits or amending.

Reply:
>  I was not sure if you prefer new commits or amending.

For PRs like this, we usually end up squashing the commits into a single one during the merge. So either way is fine. (For larger changes that touch multiple places of the codebase, I prefer multiple atomic commits, which I think make the history easier to read).